numpy/distutils/msvccompiler.py

```diff
@@ -15,3 +17,6 @@ def initialize(self, plat_name=None):
             os.environ['lib'] = environ_lib + os.environ['lib']
         if environ_include is not None:
             os.environ['include'] = environ_include + os.environ['include']
+        if platform_bits == 32:
```

Comment:
A note on why SSE2 is reguired would be helpful. Is there anyway to make this specific to compiler versions <= 1500?